doc/developer_guide/developer_guide.rst

```diff
@@ -79,4 +79,23 @@ One of the release steps failed (Partial Release)
 
     **Scenario**: Publishing of the release on Github was successfully but during the PyPi release, the upload step got interrupted.
 
-    **Solution**: Manually push the package to PyPi
\ No newline at end of file
+    **Solution**: Manually push the package to PyPi
+
+Running Tests
```

Comment:
Tthank you, for adding this to the developer guide

Reply:
🙂 Thanks for acknowledgement.